test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-Json.Tests.ps1

```diff
@@ -132,6 +132,16 @@ Describe 'ConvertFrom-Json Unit Tests' -tags "CI" {
         '[ 1 ]' | ConvertFrom-Json -NoEnumerate | ConvertTo-Json -Compress | Should -Be '[1]'
     }
 
+	It 'Properly handles a null first element' {
+		(@('5','','','','','') | convertfrom-json).count | should -be 6
+		(@('','5','','','','') | convertfrom-json).count | should -be 6
```

Comment:
Perhaps we should also have tests for `null` first elements
```
		(@($null,'5',$null,$null,$null,$null) | convertfrom-json).count | should -be 6
```

Reply:
$null will throw as inputobject is a mandatory parameter for convertfrom-json. This is really about passing an empty string to the deserializer so that it outputs a null.